kytos/core/controller.py

```diff
@@ -132,6 +132,76 @@ def enable_logs(self):
         LogManager.enable_websocket(self.api_server.server)
         self.log = logging.getLogger("controller")
 
+    @staticmethod
+    def loggers():
+        """List all logging Loggers.
+
+        Displays the Logger object, with name and logging level.
+        """
+        loggers = []
+        for name in logging.root.manager.loggerDict:
+            if "kytos" in name:
+                loggers.append(logging.getLogger(name))
+        return loggers
```

Comment:
I think a list compreehension could make the code a little more clear/concise here.
```
return [logging.getLogger(name)
        for name in logging.root.manager.loggerDict
        if 'kytos' in name]
```